### PR TITLE
Resprite of old guns

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Resprites PPSh, M60, MAC-15, Basira bolt action rifle, S&W revolver

Resprites PPSh and M60 mags

Gives S&W revolver actual in hand sprites and belt icon (it had literally none)

Resprites inhands sprites of guns above except mac15, they are the same.

All sprites by Crowford, discord tag sir.crow

# Explain why it's good for the game

Ancient sprites bad, missing icons bad.

Before:
dreamseeker_kQ1mLJNs5n

After:
dreamseeker_cBcoztKNf3

# Changelog
:cl:
fix: s&w revolver now has belt/pouch and inhand icons
imageadd: resprite of mac15, m60, ppsh, s&w revolver, basira bolt action rifle
/:cl:

### CODE REVIEW

**Comment by efzapa** [comment]
wood on the rifle is too shiny but thats just me, could make it look like the PPSh

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -2386,6 +2386,37 @@ Defined in conflicts.dm of the #defines folder.
...
+	icon_state = "m60_stock"
+	attach_icon = "m60_stock"
+	slot = "stock"
+	wield_delay_mod = WIELD_DELAY_NONE
+	flags_attach_features = NO_FLAGS
+	melee_mod = 15
+	size_mod = 0
+
+/obj/item/attachable/stock/m60/New()
+	..()
```
Comment (harryob) [inline code/modules/projectiles/gun_attachables.dm:2402]: despite this file being full of them, these new overrides are pointless, delete it

File: code/modules/projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -2386,6 +2386,37 @@ Defined in conflicts.dm of the #defines folder.
...
+	icon_state = "ppsh17b_stock"
+	attach_icon = "ppsh17b_stock"
+	slot = "stock"
+	wield_delay_mod = WIELD_DELAY_NONE
+	flags_attach_features = NO_FLAGS
+	melee_mod = 10
+	size_mod = 0
+
+/obj/item/attachable/stock/ppsh/New()
+	..()
```
Comment (harryob) [inline code/modules/projectiles/gun_attachables.dm:2417]: ditto

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]
i have eyes to see and those look nicer

**Pull request merged**